src/apps/qi-dao/helpers/qi-dao.vault.position-helper.ts

```diff
@@ -37,7 +37,7 @@ export class QiDaoVaultPositionHelper {
   constructor(
     @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
     @Inject(QiDaoContractFactory) protected readonly contractFactory: QiDaoContractFactory,
-  ) {}
+  ) { }
```

Comment:
Fix linting errors 🙏

Reply:
I believe it was VSCode which auto-formatted on save, and VSC will follow ESLint. For example, adding multiple spaces between the `{}`s will be turned into `{ }`.

Reply:
You can run the linter with `pnpm run lint:fix` to resolve all linting problems.

Or, you can install the Prettier and ESLint VSCode extensions, and those should pick up the project configs, and format properly on save.

Reply:
Running `pnpm run lint:fix` created the following patchfile: https://gist.github.com/the-emerald/096572bd8d10824be36aede65fa9f26e

with the error:

```
/home/emerald/zapper-studio/src/apps/bastion-protocol/helper/bastion-protocol.supply.token-helper.ts
  27:37  error  'T' is defined but never used  @typescript-eslint/no-unused-vars
  27:69  error  'V' is defined but never used  @typescript-eslint/no-unused-vars
```

Reply:
Relevant files have been fixed in d9d6da3